client/index.js

```diff
@@ -9,6 +9,11 @@ const Sound = require('./lib/sound.js');
 const Matrix = require('./lib/matrix.js');
 const {Eye, Plane} = require('./lib/eye.js');
 
+const Lighting = require('./lib/lighting.js');
```

Comment:
eh this is a bit inaccurate, the `require()` returns a `{components: {Lighting}}` object

Reply:
`LightingModule`?